cmake/FindrocRAND.cmake

```diff
@@ -0,0 +1,40 @@
+#=================================================================================
+# Copyright 2020-2022 Intel Corporation
+#=================================================================================
+
+if(NOT DEFINED HIP_PATH)
+  if(NOT DEFINED ENV{HIP_PATH})
+    set(HIP_PATH "/opt/rocm/hip" CACHE PATH "Path to which HIP has been installed") 
```

Comment:
This default value seems to work only for Linux. Does it make sense to have separate default values for Linux and Windows?

Reply:
Currently, ROCm is not supported on Windows ([ref](https://docs.amd.com/bundle/ROCm-Deep-Learning-Guide-v5.3/page/Troubleshooting.html#:~:text=ROCm%20is%20not%20supported%20on%20Windows.))
Hence, we've only targeted the cmake configuration for rocRAND on Linux only.